### PR TITLE
zh-translation:/docs/tasks/policy-enforcement/rate-limiting/index.md

### PR DESCRIPTION
**Pull request by luxious**
ref: https://github.com/servicemesher/istio-official-translation/issues/1122

### CODE REVIEW

**Comment by luxious** [comment]
> @luxious: The following test **failed**, say `/retest` to rerun them all:
> Test name 	Commit 	Details 	Rerun command
> lint_istio.io 	[a8f956a](https://github.com/istio/istio.io/commit/a8f956acda2e04eed9112c221d7639d3d152c30c) 	[link](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio.io/5630/lint_istio.io/1954) 	`/test lint_istio.io`

/retest

**Comment by yuer1727** [comment]
互相 review ，我的三个pr  https://github.com/istio/istio.io/issues?q=is%3Aopen+mentions%3Ayuer1727。 谢谢

**Comment by yuer1727** [comment]
> [https://github.com/istio/istio.io/issues?q=is%3Aopen+mentions%3Ayuer1727](https://github.com/istio/istio.io/issues?q=is%3Aopen+mentions%3Ayuer1727%E3%80%82)

 https://github.com/istio/istio.io/issues?q=is%3Aopen+mentions%3Ayuer1727

File: content/zh/docs/tasks/policy-enforcement/rate-limiting/index.md [hunk 1]
```diff
@@ -1,174 +1,151 @@
...
+1. 部署 [Bookinfo](/zh/docs/examples/bookinfo/) 示例应用。
 
-    The Bookinfo sample deploys 3 versions of the `reviews` service:
+    Bookinfo 部署了 3 个版本的 `reviews` 服务：
 
-    * Version v1 doesn’t call the `ratings` service.
-    * Version v2 calls the `ratings` service, and displays each rating as 1 to 5 black stars.
-    * Version v3 calls the `ratings` service, and displays each rating as 1 to 5 red stars.
+    * 版本 v1 不会调用 `ratings` 服务。
+    * 版本 v2 调用 `ratings` 服务，且为每个评价显示 1 到 5 颗黑色星星。
```
[the remark this answers is not in the document]
Comment (luxious) [reply content/zh/docs/tasks/policy-enforcement/rate-limiting/index.md]: > 评价 改为 速率评分 会不会好点？

这里是示例应用商品网站的商品评分。

**Pull request merged**